packages/core/src/actions/getTaskQueuePaused.ts

```diff
@@ -0,0 +1,20 @@
+import { getWalletId } from './getWalletId.js'
+
+import { getRelayerWithAuth } from '../utils/http.js'
+
+import { GET_TASK_QUEUE_PAUSED_ROUTE } from '../constants.js'
+import type { Config } from '../createConfig.js'
+
+export type GetTaskQueuePausedReturnType = Promise<boolean>
+
+export async function getTaskQueuePaused(
+  config: Config,
```

Comment:
Could we use the `RenegadeConfig` type to future proof this? That type encapsulates both internally and externally managed wallets.